portfolio.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel = "stylesheet" href = "css/portfolio.css"/>
+    <title>Portfolio Page</title>
+</head>
+<body>
+    <nav class="navbar">
+        <ul class="nav-links">
+            <a href="index.html" class="nav-item">Home</a>
+            <a href="about.html" class="nav-item">Bio</a>
+            <a href="experience.html" class="nav-item">Experience</a>
+            <a href="portfolio.html" class="nav-item">Portfolio Page</a>
+        </ul>
+        
+    </nav>
+    <main class="flex-main">
+        <aside class="sidebar">
+            <p id="magic" class="project-link">Magician</p>
+            <p id="claims" class="project-link">Claims.io</p>
+            <p id="techhub" class="project-link">TechHub</p>
+            <p id="fast-track" class="project-link">Fast Track</p>
+        </aside>
+        <section class="main-content">
+            <div class="image-box active">
```

Comment:
I appreciate the semantic tags used in this and other pages! see if there are others that you can use that could make your page even more accessible - https://www.w3schools.com/html/html5_semantic_elements.asp